### ISSUE TITLE
Incorrect representation of master overnight documentation build

### ISSUE THREAD

**Issue by albert-github** [CGAL/cgal#8645]
When looking at the overnight documentation builds through https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/ we see when:
- for each of the available builds (1.8.13, 1.9.6, master)
  - click the version
  - click in the left hand tree "Package Overview"
  - select the "User Manual" of the first package ("Algebraic Foundations")

We see for:

**1.8.13**
======
![Image](https://github.com/user-attachments/assets/459aee64-31f5-4344-9ae7-0128404bb95a)

**1.9.6**
======
![Image](https://github.com/user-attachments/assets/02a69175-680b-4c0c-84cc-f9a1feb4c110)

**master**
=======
![Image](https://github.com/user-attachments/assets/4f5564d0-feed-41c8-be62-49a28735d8f7)

My analysis:
========

In he HTML source of the **1.8.13** and **1.9.6** versions we see:
```
<!-- This should probably be an extrastylesheet instead of hardcoded. -->
<link href="../Manual/cgal_stylesheet.css rel="stylesheet" type="text/css">
```
and later on
```
<link href="cgal_stylesheet.css" rel="stylesheet" type="text/css">
```

Though in the **master** version we only see:
```
<link href="cgal_stylesheet.css" rel="stylesheet" type="text/css">
```

This setting is due to the, correct, setting, in BaseDoxyfile.in, of
```
HTML_EXTRA_STYLESHEET  = ${CGAL_DOC_RESOURCE_DIR}/cgal_stylesheet.css
```
but the file is not copied to the relevant package directories when copying the files from the build directories to the "display" directories (https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test). 
That the 1.8.13 and 1.9.6 versions do work is probably due to the double setting and the fact that the the css file(s) of the Manual directory is(are) copied.

**Comment by lrineau** [comment]
Fixed by https://github.com/CGAL/cgal/pull/8665.

**Comment by albert-github** [comment]
I don't think this is fixed as I still see the problem in the current overnight build: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.1-Ic-57/master/Algebraic_foundations/index.html#Chapter_Algebraic_Foundations

(Unfortunately I cannot reopen the issue).

**Comment by sloriot** [comment]
> I don't think this is fixed as I still see the problem in the current overnight build: https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.1-Ic-57/master/Algebraic_foundations/index.html#Chapter_Algebraic_Foundations
> 
> (Unfortunately I cannot reopen the issue).

The branch was not merged in master. I just did it. We'll see tomorrow.

**Comment by sloriot** [comment]
Looks good now:
https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.1-Ic-60/master/Manual/index.html

(despite other issues coming from other branches)
